src/main/main.ts

```diff
@@ -58,13 +60,15 @@ export type Client = {
   signup: (params: SignupParams) => Promise<AuthResult>
   signupWithWebAuthn: (params: SignupWithWebAuthnParams, auth?: AuthOptions) => Promise<AuthResult>
   startPasswordless: (params: PasswordlessParams, options?: Omit<AuthOptions, 'useWebMessage'>) => Promise<void>
+  startPhoneNumberRegistration: (params: StartPhoneNumberRegistrationParams) => Promise<void>
```

Comment:
I think we can hint a little more at this being MFA registration 😄 
```suggestion
  startPhoneNumberRegistration: (params: StartMfaPhoneNumberRegistrationParams) => Promise<void>
```
Otherwise, it's hard to tell the difference between `PhoneNumberVerificationParams` and `StartPhoneNumberRegistrationParams`.

Reply:
Sure, would be clearer 👍